### ISSUE TITLE
0x1 Blockchain Certificate - A lifelong achievement bookkeeper for you

### ISSUE THREAD

**Issue by tina1998612** [gitcoinco/beyondblockchainteams#2]
**Classification**
* I have an idea - am looking for teammates!

**Project Idea**
0x1 Blockchain Certificate is a certificate custody system for serializing any certificate issued for you in life from all the universities, agencies, authorities, and anyone worth recording. The under tech stack uses IPFS for file storage and Ethereum for validating the certificates.
- FB Page: https://www.facebook.com/0x1certificate/ 

**About Me**
I'm now a research scholar at Berkeley Blockchain Lab, the co-founder & CTO at [Turing Chain](turingchain.tech), and technical writer with over 3k subscribers on Medium, HackerNoon, and Coinmonks.
- Github: https://github.com/tina1998612 
- Medium: https://www.medium.com/@tina26919742 
- Linkedin: https://www.linkedin.com/in/leetingting 

**Ways to Expand**
- Target Unversity Adoptions (For example, course completion, graduation certificates, we now have over 16 universities using)
- Issue Certificates for Community Events (Events we have issued includes Angelhack, IOTA Meetup, CryptoFridayNight)

**Eligible Bounties**
TBD

**Certificate Examples**
![BASF Example](https://user-images.githubusercontent.com/19768276/59823904-7aaf0500-92e4-11e9-86a5-73bfe2523b27.png)
![AngelHackCerts_Big_Alan](https://user-images.githubusercontent.com/19768276/59823927-8f8b9880-92e4-11e9-9f95-1b967a3a4144.png)
<img width="1189" alt="Dominik" src="https://user-images.githubusercontent.com/19768276/59823988-bba71980-92e4-11e9-933a-aaf3b51d909c.png">
https://ipfs.io/ipfs/Qmd8hhbxm21YcRZNUVQsU7BHYTXMPqmez3nwN6dXcfozjS?fbclid=IwAR0XCyoVTT-zD4ME9nkpP1cqdeUT_kU7GyJTzmTPysaHQ5DrFPAVw0FkmJ0

**Comment by nanspro** [comment]
Hey :wave:  
I found this interesting. It is simple and useful but i have heard about certificates generation related projects before and so was wondering whether it is implemented before or not ? Anyway i think a good way of serializing the data will be required and then using IPFS for storage part , with a proper ui to handle all these interactions. This sounds like a good project to do :1st_place_medal: 
 
**About Me**
I am currently a CS undergrad student and been working with blockchain based applications since last year(primarily ethereum). I have never used IPFS before but i have pretty good experience with writing contracts in solidity and vyper. 

I can read about IPFS and work on that part or i can also work with the contracts section which will handle the serializing and verification part, it depends upon the other members skillset.

I have worked with a lot of projects in this ecosystem so it's hard to summarize :) so feel free to checkout my github, and [gitcoin](https://gitcoin.co/profile/nanspro)!!

**Comment by sudeepb02** [comment]
Hi @tina1998612 
Surprisingly, I've had this idea, and already developed a sample webapp for storing certificates on the blockchain. The webapp is live on the permaweb blockchain and you can find it here - 

[BlockCert - Permanent certificates on the blockchain for your achievements](https://arweave.net/M_zLrSfqvcmXEruV2756eUNROW8QP0JhUE0FVjSdLiQ)


This webapp is pretty much basic and arweave here provided the platform, but I look forward to doing amazing things, starting from scratch.

**Comment by ajmachado** [comment]
Hey @tina1998612 

I'd be very interested in joining your team. I read [https://www.cio.com/article/3291758/how-singapore-is-using-blockchain-outside-of-crypto-currencies.html]  a few days ago which describes how a university in Singapore is using blockchain to issue degrees.

I have beginners experience developing on the Ethereum platform using Solidity and have used IPFS to store images on one of my projects.

**Comment by tina1998612** [comment]
> Hey @tina1998612 👋
> I found this interesting. It is simple and useful but i have heard about certificates generation related projects before and so was wondering whether it is implemented before or not ? Anyway i think a good way of serializing the data will be required and then using IPFS for storage part , with a proper ui to handle all these interactions. This sounds like a good project to do 🥇
> 
> **About Me**
> I am currently a CS undergrad student and been working with blockchain based applications since last year(primarily ethereum). I have never used IPFS before but i have pretty good experience with writing contracts in solidity and vyper.
> 
> I can read about IPFS and work on that part or i can also work with the contracts section which will handle the serializing and verification part, it depends upon the other members skillset.
> 
> I have worked with a lot of projects in this ecosystem so it's hard to summarize :) so feel free to checkout my github, [linkedin](https://linkedin.com/in/rahul-bishnoi-159176172/) and [gitcoin](https://gitcoin.co/profile/nanspro)!!

Yeah we now have the ipfs and Ethereum smart contract code base available (though haven't fully tested yet). Will you be interested in helping to improve and test the contract?

**Comment by tina1998612** [comment]
> Hi @tina1998612
> Surprisingly, I've had this idea, and already developed a sample webapp for storing certificates on the blockchain. The webapp is live on the permaweb blockchain and you can find it here -
> 
> [BlockCert - Permanent certificates on the blockchain for your achievements](https://arweave.net/M_zLrSfqvcmXEruV2756eUNROW8QP0JhUE0FVjSdLiQ)
> 
> This webapp is pretty much basic and arweave here provided the platform, but I look forward to doing amazing things, starting from scratch.

Wow amazing!! It'd be super cool if we can build together! 😃 
What's your email address? We can discuss over there:)

**Comment by tina1998612** [comment]
> Hey @tina1998612
> 
> I'd be very interested in joining your team. I read https://www.cio.com/article/3291758/how-singapore-is-using-blockchain-outside-of-crypto-currencies.html a few days ago which describes how a university in Singapore is using blockchain to issue degrees.
> 
> I have beginners experience developing on the Ethereum platform using Solidity and have used IPFS to store images on one of my projects.

Do you have experience as well in building the frontend part for the webapp or mobile app that can connect to web3? Thanks!

**Comment by nanspro** [comment]
@tina1998612  Sure :)
I am currently busy with some tasks which i have to complete by monday. So i will go through the contracts (and will update them accordingly) from 25th if that's okay with you.

**Comment by ajmachado** [comment]
> > Hey @tina1998612
> > I'd be very interested in joining your team. I read https://www.cio.com/article/3291758/how-singapore-is-using-blockchain-outside-of-crypto-currencies.html a few days ago which describes how a university in Singapore is using blockchain to issue degrees.
> > I have beginners experience developing on the Ethereum platform using Solidity and have used IPFS to store images on one of my projects.
> 
> Do you have experience as well in building the frontend part for the webapp or mobile app that can connect to web3? Thanks!

I can help with the web3Js, but I am not strong at frontend development. I can also help test the smart contracts and IPFS storage.

**Comment by thesachinmittal** [comment]
Greetings @tina1998612 

I have been introduced to several platforms on this idea. Only a few of them are able to penetrate the market. And your platform has done a commendable job. Congratulations on that. 

I am quite interested in working along with your team. Currently, I am enrolled in the consensys developer program which has provided me with an enriching experience in ethereum development. Also, I have previous experience with ipfs and ethereum projects.

I am especially inclined towards web3 and improving the contracts.

For my previous work(relevant to this issue), please go through my Github projects. (Justice league, Kryptonite, Krishi Mitra, ethereum-ipfs).

**Comment by sanchaymittal** [comment]
Greetings @tina1998612

I am very much interested in working along. This is a robust use case of blockchain technology. I have previously tinkered with this idea, using hyperledger. Documented in github repo of Hyperledger Iroha. (Working as a summer intern for Linux foundation and hyperledger)

Looking forward to hearing from you.
Mail id - sanchaymittal@gmail.com

**Comment by sudeepb02** [comment]
Hi @tina1998612 
You can reach me at sudeepbiswas02@gmail.com

**Comment by zheli** [comment]
What kind of teammate are you looking for? What competencies are required?

**Comment by vs77bb** [comment]
@tina1998612 Looks great! Are [you in Discord?](https://discord.gg/sqWtCbj) I'm making a channel for you all to discuss further if you'd like there! 

As the prizes get posted, curious to hear if any of them overlap with what you'd like to build! I could see Raiden / Arweave's prizes potentially fitting with what you're doing 🙁

**Comment by tina1998612** [comment]
> Raiden / Arweave

Thanks for your suggestions and kind support! This one best fits us: https://github.com/raiden-network/hackathons/issues/1

**Comment by calchulus** [comment]
Hey this sounds pretty neat! Let me know what I can do to help!
